cmd/kpod/stop.go

```diff
@@ -50,19 +50,18 @@ func stopCmd(c *cli.Context) error {
 	if err != nil {
 		return errors.Wrapf(err, "could not update list of containers")
 	}
-	hadError := false
+	var lastError Error
 	for _, container := range c.Args() {
 		cid, err := server.ContainerStop(container, stopTimeout)
 		if err != nil {
-			hadError = true
-			logrus.Error(err)
+			if lastError != nil {
+				logrus.Error(lastError)
```

Comment:
I think the last error will not be printing to logrus (because it's being printed by the cli.App), but probably to stderror - might want to do that here as well